src/libraries/pretest.proj

```diff
@@ -4,7 +4,9 @@
   <PropertyGroup>
     <TraversalGlobalProperties>BuildAllProjects=true</TraversalGlobalProperties>
     <NETCoreAppFrameworkIdentifier>$(TargetFrameworkIdentifier)</NETCoreAppFrameworkIdentifier>
+  </PropertyGroup>
 
+  <PropertyGroup Condition="'$(ContinuousIntegrationBuild)' != 'true'">
     <!-- Create an intermediate runsettings file to enable VSTest discovery. -->
     <EnableRunSettingsSupport>true</EnableRunSettingsSupport>
```

Comment:
Why do we need to declare this here as well, isn't Directory.Build.props imported into this project build? That way we shouldn't need to condition this either.

Reply:
pretest.proj is no `$(TestProject)`: https://github.com/dotnet/runtime/pull/36126/files#diff-d3bedba39ce6da3e37ca0db83d7ef035R328

Reply:
Ah, I missed that.